app/src/ui/repository-settings/git-ignore.tsx

```diff
@@ -23,6 +37,9 @@ export class GitIgnore extends React.Component<IGitIgnoreProps, {}> {
           for more information about the file format, or simply ignore a file by
           right clicking on it in the uncommitted changes view.
         </p>
+
+        {this.renderBranchWarning()}
```

Comment:
`{this.props.isDefaultBranch && <p>...</p>}`?

Reply:
That would return `false` instead of `null` in the negative case. Would that render differently?

Reply:
[Nope](https://reactjs.org/docs/jsx-in-depth.html#booleans-null-and-undefined-are-ignored):

> `false`, `null`, `undefined`, and `true` are valid children. They simply don’t render.

Reply:
Thanks for that @j-f1. Still, six of one, half dozen of the other. I feel this looks cleaner and the same pattern is used in other UI elements.